lib/rewriter.js

```diff
@@ -3,10 +3,11 @@
 const assert = require('assert');
 const ts = require('typescript');
 const tsMorph = require('ts-morph');
+const {addAllImports} = require('./imports.js');
```

Comment:
Nit: if you move this to the bottom then the diff is simpler. (I.e. then this PR doesn't move it around.)